base/compiler/ssair/inlining.jl

```diff
@@ -428,14 +427,14 @@ function ir_inline_item!(compact::IncrementalCompact, idx::Int, argexprs::Vector
         just_fixup!(inline_compact)
         compact.result_idx = inline_compact.result_idx
         compact.active_result_bb = inline_compact.active_result_bb
-        for i = 1:length(pn.values)
-            isassigned(pn.values, i) || continue
-            v = pn.values[i]
-            if isa(v, SSAValue)
-                compact.used_ssas[v.id] += 1
-            end
-        end
```

Comment:
Here, if `length(pn.edges) != 1`, we call `insert_node_here!` on `NewInstruction(pn...`, which does this same incrementing. In the case where we just return `pn.values[1]`, I'm not sure if there are paths that result in this getting counted twice, but we can at least make this change for now